numba/tests/test_numpy_support.py

```diff
@@ -422,5 +423,69 @@ def check_arr(arr):
         check_arr(arr.reshape((2, 2, 3, 2)).T[:, ::3, ::2, ::5])
 
 
+class TestSupportOverloads(TestCase):
```

Comment:
This class should go into `numba/tests/test_np_functions.py` this module is for testing the module `numba.np.numpy_support`.